run/GCHP/runScriptSamples/operational_examples/harvard_cannon/gchp.run

```diff
@@ -28,11 +28,9 @@ export OMPI_MCL_btl=openib
 # Run GCHP and evenly distribute tasks across nodes
 NX=$( grep NX GCHP.rc | awk '{print $2}' )
 NY=$( grep NY GCHP.rc | awk '{print $2}' )
-coreCount=$(( ${NX} * ${NY} ))
-planeCount=$(( ${coreCount} / ${SLURM_NNODES} ))
-if [[ $(( ${coreCount} % ${SLURM_NNODES} )) > 0 ]]; then
-	${planeCount}=$(( ${planeCount} + 1 ))
-fi
+let coreCount=$(( ${NX} * ${NY} ))
+let planeCount=$(( ${coreCount} / ${SLURM_NNODES} ))
+[[ $(( ${coreCount} % ${SLURM_NNODES} )) > 0 ]] && let planeCount+=1
```

Comment:
This is a bit hard to parse and understand. Could we stick with old school style and do planeCount=$(( ${planeCount} + 1 ))?